TWLight/applications/views.py

```diff
@@ -1147,7 +1206,11 @@ def get_queryset(self):
             status=Application.APPROVED, editor__isnull=False
         ).exclude(editor__user__groups__name="restricted")
 
-        partner_list = Partner.objects.filter(applications__in=base_queryset).distinct()
+        partner_list = (
+            Partner.objects.filter(applications__in=base_queryset)
+            .exclude(authorization_method__in=[Partner.PROXY, Partner.BUNDLE])
```

Comment:
It might actually better to `.filter()` for an `authorization_method` of `Partner.EMAIL` or `Partner.CODES`, since `.LINK` also shouldn't be something this page should care about.